### PR TITLE
Wget option ordering

### PR DESCRIPTION
**Pull request by kamirr**
Patch from #32

### CODE REVIEW

**Review by mfranciszkiewicz — APPROVED** [review]

File: installer.sh [hunk 1]
```diff
@@ -122,14 +122,14 @@ EOF
 
 check_terms_accepted() {
     local _tagdir="$1"
-    local _tags=${@:2}
+    local _tags="${@:2}"
```
Comment (mfranciszkiewicz) [inline installer.sh:125]: this is no longer an array, which is converted to string

**Review by mfranciszkiewicz — APPROVED** [review]
I'm unsure though why the reversing the order is better

**Comment by kamirr** [comment]
> I'm unsure though why reversing the order is better

That's the order defined in the manpage, and the author reported a failure related to that.

**Comment by kamirr** [comment]
Sorry for re-requesting the review again, the commit got unsigned again after I rebased it.

**Review by mfranciszkiewicz — APPROVED** [review]

**Pull request merged**